heron/config/src/yaml/BUILD

```diff
@@ -14,7 +14,11 @@ filegroup(
 
 filegroup(
     name = "conf-yaml",
-    srcs = glob(["conf/**/*.yaml"] + ["conf/**/*.aurora"] + ["conf/**/*.sh"]),
+    srcs = glob(["conf/**/*.yaml"] 
```

Comment:
nit: suggest moving "[conf/**.*.yaml"] to next line, and suggest sorting them.

Reply:
will do